includes/api/class-wc-admin-rest-admin-notes-controller.php

```diff
@@ -300,19 +317,64 @@ public function prepare_item_for_response( $data, $request ) {
 	 * @return array
 	 */
 	public function get_collection_params() {
-		$params            = array();
-		$params['context'] = $this->get_context_param( array( 'default' => 'view' ) );
-		$params['type']    = array(
-			'description'       => __( 'Type of note.', 'woocommerce-admin' ),
+		$params             = array();
+		$params['context']  = $this->get_context_param( array( 'default' => 'view' ) );
+		$params['order']    = array(
+			'description'       => __( 'Order sort attribute ascending or descending.', 'woocommerce-admin' ),
 			'type'              => 'string',
-			'enum'              => WC_Admin_Note::get_allowed_types(),
+			'default'           => 'desc',
+			'enum'              => array( 'asc', 'desc' ),
```

Comment:
for these string based args, with pre-defined options in an array - in a recent PR in Woo core @claudiosanches had me add in `'sanitize_callback' => 'sanitize_key'` i.e. https://github.com/woocommerce/woocommerce/blob/master/includes/api/wc-blocks/class-wc-rest-blocks-products-controller.php#L326

Which if I followed the trail properly would allow someone passing in `ASC` to be downcased ( but do correct me if I'm wrong Claudio! )

Reply:
Hmm, I tried this with `sanitize_key` and it didn't work with any uppercase characters.  It didn't prevent anything from working here either so I'm not opposed to replacing this if there are added benefits. /cc @claudiosanches

Reply:
In fact there's no need, since should only expect results like listed in `enum`.